### DISCUSSION TITLE
Restyling

### DISCUSSION THREAD

**Discussion by Ajaymamtora** [photoprism/photoprism#1227] [Q&A] [answered]
Hi, is there a possibility for me to try to restyle the app in my own way? Potentially using a material library?

**Comment by lastzero** [comment, marked as answer]
There's no easy way to do this right now. We're using Vuetify which is based on Material Design.

**Reply by Ajaymamtora** [reply]
Is there a plan to upgrade to material design 2 / material next when that releases?

**Reply by lastzero** [reply]
That depends on our resources, and those depend on our funding. You help us, we help you!

**Reply by Ajaymamtora** [reply]
Makes sense, thanks for the replies 😄
